src/go/MONIT/monit.go

```diff
@@ -978,13 +992,35 @@ func main() {
 	}
 	dbID := dbid
 	var database, dbtype string
+	if token == "" {
+		log.Fatalf("Please provide valid token")
+	}
 	if strings.Contains(q, "stats") {
-		// per our discussion with MONIT team
-		// https://cern.service-now.com/service-portal?id=ticket&table=u_request_fulfillment&n=RQF1647972
+		// per our discussion with MONIT team (RQF1647972)
 		// we only need to use some dbid for _stats ES API, here I use
-		// 9573, the id of monit_prod_cms-es-size_raw_elasticsearch collection
-		dbid = 9573
-		url = fmt.Sprintf("%s/api/datasources/proxy/%d/_stats/store", url, dbid)
+		// 9573, the id of monit_prod_cms-es-size_raw_elasticsearch datasource, and 9582
+		// the id of monit_prod_condor_agg_metric datasource
+		short_term_dbid := 9573
+		long_term_dbid := 9582
+		short_term_url := fmt.Sprintf("%s/api/datasources/proxy/%d/monit_prod*/_stats/store", url, short_term_dbid)
+		long_term_url := fmt.Sprintf("%s/api/datasources/proxy/%d/monit_prod*/_stats/store", url, long_term_dbid)
+		getStats(short_term_url, long_term_url, t, short_term_dbid, long_term_dbid, database, q, esapi, idx, limit, verbose, stompConfig, inject, hdfs)
+		// obtain HDFS records if requested
+		if hdfs != "" {
+			records := hdfsDump(hdfs, verbose)
+			injectRecords(stompConfig, records, verbose, inject)
+		}
+		if verbose > 1 {
+			log.Println("short_term_url ", short_term_url)
```

Comment:
If you'll add StatsParams structure then you can define its Print() method to do what this block represents, e.g.
```
type StatsParams struct {
...
}
func (s *StatsParams) Print() {
   log.Println("short_term_url ", s.LongUrl)
   ...
}
```